README.md

```diff
@@ -0,0 +1,67 @@
+# Rootless Podman Homemanager Module
+
+A Home-manager module to generate systemd services for Rootless Podman Pods from a Nixos `virtualisation.oci-containers` / docker-compose like style.
+
+__Note:__ Currently it is only supported to use containers which are part of a pod.
+
+## Installation
+```
+{
+  inputs.rootless-podman-homeManager.url = "github:FabianKoehnen/rootless-podman-homemanager";
+
+  outputs = { self, nixpkgs, rootless-podman-homeManager }: {
+    # change `yourhostname` to your actual hostname
+    nixosConfigurations.yourhostname = nixpkgs.lib.nixosSystem {
+      # customize to your system
+      system = "x86_64-linux";
+      modules = [
+        ./configuration.nix
+        home-manager.sharedModules = [
+            inputs.rootless-podman-homeManager.homeManagerModules.default
```

Comment:
`inputs` will not be defined here. (you'd need to 'capture' the function arguments with `outputs = inputs@{...}:`.

Or you can just use the `rootless-podman-homeManager` argument directly.

```suggestion
  outputs = { self, nixpkgs, rootless-podman-homeManager }: {
    # change `yourhostname` to your actual hostname
    nixosConfigurations.yourhostname = nixpkgs.lib.nixosSystem {
      # customize to your system
      system = "x86_64-linux";
      modules = [
        ./configuration.nix
        home-manager.sharedModules = [
            rootless-podman-homeManager.homeManagerModules.default
```